jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -548,10 +548,12 @@ public int countStackFrames() {
 /*[ELSE]*/
 @Deprecated
 /*[ENDIF]*/
+/*[IF !Java11]*/
```

Comment:
The above ` @Deprecated` statement and javadoc also needs to be removed for Java 11. Similarly for stop().